third_party/BUILD

```diff
@@ -18,6 +18,7 @@ filegroup(
         "//third_party/forked/etcd237/pkg/fileutil:all-srcs",
         "//third_party/forked/etcd237/wal:all-srcs",
         "//third_party/forked/golang/expansion:all-srcs",
+        "//third_party/forked/golang/go/types:all-srcs",
```

Comment:
Should reference upstream bug so it's clear when this fork can be removed

Reply:
I reference it in the import statement-- don't comments in BUILD files get removed by hack/update-bazel?

Reply:
Ah, didn't scroll to the README, was looking at the commit body. ACK